buildshiprun/handler_test.go

```diff
@@ -165,12 +95,51 @@ func Test_GetImageName(t *testing.T) {
 
 func Test_ValidImage(t *testing.T) {
```

Comment:
I have tested the regex for the below cases [here](https://regex101.com/r/wM3Svf/1)